### PR TITLE
Catch space key inconsistency in IE10.

### PR DESCRIPTION
**Pull request by cdata**
/cc @notwaldorf

### CODE REVIEW

File: test/basic-test.html [hunk 1]
```diff
@@ -161,6 +161,14 @@
       expect(keys.keyCount).to.be.equal(1);
     });
 
+    suite('edge cases', function() {
+      test('knows that `spacebar` is the same as `space`', function() {
```
Comment (notwaldorf) [inline test/basic-test.html]: 😒

**Comment by notwaldorf** [comment]
lgtm
